Index: FileHandler.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- FileHandler.java	(date 1593585895489)
+++ FileHandler.java	(date 1593585895489)
@@ -0,0 +1,24 @@
+package source.util;
+
+import java.io.File;
+
+public class FileHandler {
+
+    private File my_file;
+
+    static boolean read_file(String path){
+        // TODO : read from exiting file.
+
+
+        // Hint : please provide seprate load function to respective algorithms , to get the file data.
+    }
+
+    static boolean write( String path ){
+        //TODO: create if not exist and write.
+
+        // Hint : please provide seprate load function to respective algorithms , to get the file data.
+    }
+
+
+
+}
